Add tap-to-scroll option

di72nn opened the pull request:
#168.

Replaces #266.

Same note applies:

The "sides" are limited by:
 * height: (0.25; 0.75) scrollView's height;
 * width: left is [0; 0.3) of scrollView's width and right is (0.7; 1].

For some reason touch events are not consumed: if you tap on a link it will both scroll and open link menu.

di72nn commented:
Is there any problem with it?
I got the impression that the feature would be very useful for E-Ink users, but would not affect anyone who does not enable it. The implementation is somewhat poor (no UI buttons (are they needed?) and the event consumption issue (nothing serious)), but it is better than nothing.

P. S.: Now that I think about it, maybe I should also add "scroll with volume buttons" option or something like that.

Strubbl commented:
Sorry, i am not interested in this feature at all. That's why i did not review yet. 

The scroll with volume buttons option might be interesting for users without e-ink display, too.

tcitworld commented:
@di72nn This would be great !
> P. S.: Now that I think about it, maybe I should also add "scroll with volume buttons" option or something like that.

di72nn commented:
Added an option to scroll with volume buttons. Also added a "smooth scrolling" checkbox. Is it just me, or smooth scrolling with volume buttons is not always smooth (maybe threading issue or something else)?
